v03_pipeline/lib/tasks/validate_callset.py

```diff
@@ -0,0 +1,149 @@
+import hail as hl
+import luigi
+import luigi.util
+
+from v03_pipeline.lib.misc.callsets import additional_row_fields
+from v03_pipeline.lib.misc.validation import (
+    validate_allele_type,
+    validate_expected_contig_frequency,
+    validate_imported_field_types,
+    validate_imputed_sex_ploidy,
+    validate_no_duplicate_variants,
+    validate_sample_type,
+)
+from v03_pipeline.lib.model import CachedReferenceDatasetQuery
+from v03_pipeline.lib.model.environment import Env
+from v03_pipeline.lib.paths import (
+    cached_reference_dataset_query_path,
+    imported_callset_path,
+    sex_check_table_path,
+)
+from v03_pipeline.lib.tasks.base.base_loading_run_params import BaseLoadingRunParams
+from v03_pipeline.lib.tasks.base.base_update import BaseUpdateTask
+from v03_pipeline.lib.tasks.files import CallsetTask, GCSorLocalTarget, HailTableTask
+from v03_pipeline.lib.tasks.reference_data.updated_cached_reference_dataset_query import (
+    UpdatedCachedReferenceDatasetQuery,
+)
+from v03_pipeline.lib.tasks.write_imported_callset import WriteImportedCallsetTask
+from v03_pipeline.lib.tasks.write_sex_check_table import WriteSexCheckTableTask
+
+
+@luigi.util.inherits(BaseLoadingRunParams)
+class ValidateCallsetTask(BaseUpdateTask):
```

Comment:
is there anything in here that wasn't pulled from `WriteImportedCallsetTask`?

Reply:
nope!

Reply:
lifted line by line.